modules/core/src/main/scala/org/scalasteward/core/git/package.scala

```diff
@@ -21,17 +21,18 @@ import org.scalasteward.core.repoconfig.CommitsConfig
 import org.scalasteward.core.update.show
 
 package object git {
-  def branchFor(update: Update): Branch =
-    Branch(s"update/${update.name}-${update.nextVersion}")
+  def branchFor(update: Update, baseBranch: Branch): Branch =
+    Branch(s"update/${baseBranch.name}/${update.name}-${update.nextVersion}")
```

Comment:
`branchFor` is used in `vcs.listingBranch` to create the `head` parameter for `VCSApiAlg#listPullRequests` which is used to find already existing Scala Steward PRs. If we change this for PRs that target the default branch, Scala Steward will not find older PRs it already created and will create new ones. I guess this could result in thousands new but duplicated PRs by @scala-steward.

Can we make this optional and only pass it if `baseBranch` is not the default branch?

Reply:
sure! i'll push a fix soon:)